cmd/api/src/database/migration/migrations/schema.sql

```diff
@@ -123,8 +123,8 @@ CREATE SEQUENCE IF NOT EXISTS asset_group_selectors_id_seq
 ALTER SEQUENCE asset_group_selectors_id_seq OWNED BY asset_group_selectors.id;
 
 CREATE TABLE IF NOT EXISTS asset_groups (
-    name text,
-    tag text,
+    name text UNIQUE NOT NULL,
+    tag text UNIQUE NOT NULL,
```

Comment:
I don't believe we would want to change this since it's the original migration which will have already been applied. We should only make changes in new migrations.

Reply:
Resolved
https://github.com/SpecterOps/BloodHound/pull/665#discussion_r1657562087